certification/pyxis/layers.go

```diff
@@ -34,6 +34,7 @@ func (p *pyxisEngine) CheckRedHatLayers(ctx context.Context, layerHashes []crane
 		return nil, err
 	}
 
+	log.Tracef("request to pyxis is %s?filter=%s", getPyxisUrl("images"), pyxisQuery)
```

Comment:
We should probably log the URL in the request and not trust the code we built. Like the rest of the pyxis calls.
```suggestion
	log.Debugf("URL is: %s",  req.URL)
```

Reply:
agreed 👍